tck/optional/src/main/java/org/eclipse/microprofile/metrics/test/optional/MPMetricBaseMetricsTest.java

```diff
@@ -303,28 +303,28 @@ public void testGcTimeMetrics() {
         String data = given().header(wantPromMetricsFormat).get("/metrics?scope=base").asString();
 
         Map<String, MiniMeta> baseNames = getExpectedMetadataFromXmlFile(MetricRegistry.BASE_SCOPE);
-        MiniMeta gcCountMetricMeta = baseNames.get("gc.time");
-        Set<String> expectedTags = gcCountMetricMeta.tags.keySet();
+        MiniMeta gcTimeMetricMeta = baseNames.get("gc.time");
+        Set<String> expectedTags = gcTimeMetricMeta.tags.keySet();
 
         String[] lines = data.split("\n");
 
         boolean found = false;
         for (String line : lines) {
             // explicitly check for the metric line wth value (i.e. the use of `{`)
-            if (line.contains("gc_time_seconds_total{")) {
-                final Pattern gcTimeTotalPattern = Pattern.compile("(gc_time_seconds_total\\{.*?\\}) (\\d+\\.\\d+)");
-                assertThat("Line format should be gc_time_seconds_total\\{.*?\\} \\d+\\.\\d+",
+            if (line.contains("gc_time_seconds{")) {
+                final Pattern gcTimeTotalPattern = Pattern.compile("(gc_time_seconds\\{.*?\\}) (\\d+\\.\\d+)");
+                assertThat("Line format should be gc_time_seconds\\{.*?\\} \\d+\\.\\d+",
                         gcTimeTotalPattern.matcher(line).matches());
 
                 final String metricID = gcTimeTotalPattern.matcher(line).replaceAll("$1");
-                final String tags = metricID.replaceAll("^gc_time_seconds_total\\{", "").replaceAll("\\}$", "");
+                final String tags = metricID.replaceAll("^gc_time_seconds\\{", "").replaceAll("\\}$", "");
 
                 for (String expectedTag : expectedTags) {
                     assertThat("The metric should contain a " + expectedTag + " tag", tags,
                             containsString(expectedTag + "="));
                 }
                 final String value = gcTimeTotalPattern.matcher(line).replaceAll("$2");
-                Assert.assertTrue("gc.time.seconds.total value should be numeric and not negative",
+                Assert.assertTrue("gc.time value should be numeric and not negative",
```

Comment:
gc.time.seconds value... ?

Reply:
The use of periods seemed like it was referring to its original metric name which would be `gc.time`. Or we could switch it to `gc_time_seconds` to use the time unit.

Reply:
ok, your way makes sense.